tests/BUILD.bazel

```diff
@@ -8,7 +8,7 @@ sh_test(
         "//nixpkgs:srcs",
         "//tests/invalid_nixpkgs_package:srcs",
         "@coreutils_static//:bin",
-        "@nix_2_7//:bin",
+        "@nix_2_10//:bin",
```

Comment:
Any reason for thins change, except general maintenance ?

Reply:
Yes, version 2.7 (and 2.8 and 2.9) are no longer available in the more recent nixpkgs revision (unstable). So we need to bump the version.